openedx_webhooks/info.py

```diff
@@ -161,6 +163,12 @@ def is_committer_pull_request(pull_request: PrDict) -> bool:
     if "repos" in commit_rights:
         if repo in commit_rights["repos"]:
             return True
+    if "branches" in commit_rights:
+        for access_branch in commit_rights["branches"]:
+            if access_branch.endswith("*") and branch.startswith(access_branch[:-1]):
```

Comment:
OK! The fnmatch module is useful for these kinds of patterns, but this will work.